packages/editor/src/plugins/components/CredibilityIndicators.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+// @ts-ignore
```

Comment:
If it's a linter error use ```// tslint:disable-next-line```.

```// @ts-ignore``` is a compiler directive (not linter) and this one disables the whole compiler reporting errors here, meaning that if styles-components fails to import at all, we're gonna know only at runtime instead of compile time.

@dankins @jorgelo @nickreynolds  : FYI